xbmc/win32/Win32DelayedDllLoad.cpp

```diff
@@ -70,5 +70,5 @@ FARPROC WINAPI delayHookFailureFunc (unsigned dliNotify, PDelayLoadInfo pdli)
 }
 
 // assign hook functions
-PfnDliHook __pfnDliNotifyHook2 = delayHookNotifyFunc;
-PfnDliHook __pfnDliFailureHook2 = delayHookFailureFunc;
```

Comment:
@Memphiz @Paxxi 
Sorry I don't know myself. Normally I wait for response, but I was tricked by my own title. I also thought that I only moved the sources and it couldn't nothing. I totally forgot about that I should wait for your response @Paxxi.
I even forgot to squash the commits. I really apologize for my rush! Next time I will do it better. I guess I worked to much yesterday on Kodi and my brain was tired :-D